src/v0/steward.ts

```diff
@@ -270,12 +270,17 @@ export function handleLogBuy(event: LogBuy): void {
   let wildcardOwner = patron.id;
   let wildcardTimeAcquired = txTimestamp;
 
+  let eventParamsString = "['" + tokenIdString  + "', '" + event.params.owner.toHexString()  + "', '" + event.params.price.toString() + "']";
```

Comment:
Hmm, I wonder if it is better to use the `'` on the outer level, and `"` on the inner... I guess We'll only find out later if it is better. No changes required.

Reply:
I see what you mean, might be true. We'll find out 👍